app/controllers/application_controller.rb

```diff
@@ -26,34 +26,6 @@ class ApplicationController < ActionController::Base
 
   private
 
-  def meta_hash_for(results)
```

Comment:
Wondering if there was a reason for putting this here.

Nothing related in a comment that introduced this code: https://github.com/berkmancenter/lumendatabase/commit/49895f258a10ec205a1bab5365bbe5b1e1ced845.

Hopefully no reason :v:

Reply:
I had the same thought process when I moved it, lol. Couldn't find anywhere else it was used, or any reason for it to be polluting the top-level namespace. Tests still work. :shipit: